### PR TITLE
Make http logs match ECS compatible in Fluent-bit

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

Supports [ticket 2123](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2123) and the overall goal of pushing our app logs to 1Team's ElasticSearch/Kibana stack.

update fluent-bit version

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-208

File: openshift/fluent-bit.cm.yaml [hunk 1]
```diff
@@ -68,13 +85,17 @@ objects:
           # we can also send tag as a header
           #header_tag  app
 
+        # send http level logs (with tag: 'app.http') to AWS Opensearch
         # [OUTPUT]
-        #   # send logs to AWS Opensearch service
+        #   Name http
+        #   Match app.http
+
```
Comment (MichaelChung123) [inline openshift/fluent-bit.cm.yaml:92]: can we remove this white space?

**Review by MichaelChung123 — APPROVED** [review]
LGTM other than that one extra whitespace

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-208

**Review by jujaga — APPROVED** [review]

**Review by loneil — APPROVED** [review]

**Pull request merged**